### ISSUE TITLE
The phoebe pod model has been replaced with a duplicate of the proteus pod

### ISSUE THREAD

**Issue by uncertifiedengineer** [post-kerbin-mining-corporation/NearFutureSpacecraft#168]
The stats names and descriptions are still different however the models are the same.

**Comment by uncertifiedengineer** [comment]
nothing seems too out of the ordinary, I did install everything correctly, appeared only after the recent updates but i'm unsure myself.

**Comment by ChrisAdderley** [comment]
Fixed in 1.4.5
